### ISSUE TITLE
aaline may not be drawn on surface if a coordinate is 0

### ISSUE THREAD

**Issue by teodor440** [pygame/pygame#731]
**Aaline won't render horizontal/vertical lines on the first column/line of a surface**

It wouldn't work to draw the following vertical line:
```
screen_width, screen_height = (600, 400)
screen = pygame.display.set_mode((screen_width, screen_height))
pygame.draw.aaline(screen, (255, 255, 255), (0, 1), (0, 150))
pygame.display.update()
```

But if I try to draw it to the second column it will work:
```pygame.draw.aaline(screen, (255, 255, 255), (1, 1), (1, 150))```

Also I saw that it would work partially for something like `pygame.draw.aaline(screen, (255, 255, 255), (1, 1), (0, 150))` drawing one dot on the second column (actually it is the line ignoring the first column)
![dot](https://user-images.githubusercontent.com/14222938/51057575-b257a980-15ee-11e9-936a-660622ebb607.png)

I'm using Ubuntu 18.04.1 LTS

**Comment by charlesej** [comment]
This issue appears to the be same as #732. This one can be closed and #732 can be used to track the issue.

**Comment by charlesej** [comment]
This issue has been resolved by PR #802.
